client/components/data/query-reader-follows/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFollows } from 'state/reader/follows/actions';
+
+class QueryReaderFollows extends Component {
```

Comment:
so this component would be `const QueryReaderFollows = dispatchOnMount( [ "requestFollows" ] )` or something